lib/travis/build/script/go.rb

```diff
@@ -103,6 +108,13 @@ def uses_make
             '-f GNUmakefile || -f makefile || -f Makefile || -f BSDmakefile'
           end
 
+	  def uses_15_vendoring
+            if (go_version < '1.5' || go_version == 'go1')
```

Comment:
Indeed, we already use `Gem::Version` in Elixir. See https://github.com/travis-ci/travis-build/blob/c4c406ff259c47fc5d60a20a1c55482ab3b49727/lib/travis/build/script/elixir.rb#L67-L69